app/mixins/connection-type.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember';
+
+const {Mixin, inject, computed} = Ember;
+
+export default Mixin.create({
+	fastboot: inject.service(),
+	connection: computed('fastboot.isFastBoot', function () {
+		if (!this.get('fastboot.isFastBoot')) {
+			return navigator.connection || navigator.mozConnection || navigator.webkitConnection;
+		} else {
```

Comment:
I'd remove this `else`

Reply:
Does computed props defaults to false if there is no return?

Reply:
i'd assume that it'll be equal to `undefined`